manifests/client.pp

```diff
@@ -46,17 +46,14 @@
 # Copyright (C) 2012 Mike Arnold, unless otherwise noted.
 #
 class snmp::client (
-  $snmp_config        = $snmp::params::snmp_config,
-  $ensure             = $snmp::params::ensure,
```

Comment:
@ekohl Oops.  I seem to have fat fingered here and nuked a few lines I didn't mean to!